release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3623,3 +3623,19 @@ link:https://access.redhat.com/solutions/6963078[{product-title} 4.6.59 containe
 ==== Updating
 
 To update an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-6-60"]
+=== RHBA-2022:5572 - {product-title} 4.6.60 bug fix update
+
+Issued: 2022-07-21
+
+{product-title} release 4.6.59, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:5572[RHBA-2022:5572] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5571[RHBA-2022:5571] advisory.
```

Comment:
Don't think it includes security updates because ther eisn't an RHSA advisory :D

Reply:
Whoops! Thanks! Updated.